axual-webclient/groups_data.go

```diff
@@ -118,6 +139,7 @@ type GroupRequest struct {
 	EmailAddress interface{} `json:"emailAddress,omitempty"`
 	PhoneNumber  interface{} `json:"phoneNumber,omitempty"`
 	Members      []string    `json:"members,omitempty"`
+	Managers     []string    `json:"managers"`
```

Comment:
Is it required for omitempty for managers too?
```suggestion
	Managers     []string    `json:"managers, omitempty"`
```

Reply:
This is intentional. with `omitempty` we don't include it in the payload. We need to include empty array in the payload, because this is how API works with managers. Empty array means to delete existing managers. If managers are not sent, it does not delete the managers.